### PR TITLE
[DOCS-7499] Add Preconfigured workflows for APS 24.1

### PR DESCRIPTION
**Pull request by aritrihore**

### CODE REVIEW

File: process-services/latest/using/workflows.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+5. Enter the following details as per your requirements.
+
+      * Enter the **Name** – The name of your tenant, for example, xxx.
+      * Specify the **Maximum allowed users in this tenant** - This is set at 10 bBy default, it is set at 10.
+
+6. Select the **Deploy preconfigured workflows** checkbox.
+
+7. Add the details about the tenant owner including **First name**, **Last name**, **Email**, and **Password**. 
+
+      You can also assign a tenant owner from the list of created users.
```
Comment (AyushSrivastavaHyland) [inline process-services/latest/using/workflows.md:37]: we are assigning the workflow owner from the list of created users

File: process-services/latest/using/workflows.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+1. Log in to Alfresco Process Services.
+
+2. Select **Identity Management**.
+
+3. Select **Organization**.
+
+4. Select your tenant from the drop-down list.
+
+5. Select **Create group**. Name your group and click **Save**.
```
Comment (AyushSrivastavaHyland) [inline process-services/latest/using/workflows.md:95]: after selecting your tenant a "Workflow group" will be visible be default to make the other users to view the worklfow we need to add those users in this group

File: process-services/latest/using/workflows.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+      * Specify the **Maximum allowed users in this tenant** - This is set at 10 bBy default, it is set at 10.
+
+6. Select the **Deploy preconfigured workflows** checkbox.
+
+7. Add the details about the tenant owner including **First name**, **Last name**, **Email**, and **Password**. 
+
+      You can also assign a tenant owner from the list of created users.
+
+8. Click **Save**.
+
```
Comment (AyushSrivastavaHyland) [inline process-services/latest/using/workflows.md:40]: **## To Change the owner of PreConfigured_workflows**

first the owner of the PreConfigured workflow needs to manually delete the workflow app and then the owner can be changed by going into tenant section click on edit tenant and add a new user or select from the exisiting user now after saving this user will be the owner of the workflow in the current tenant

**Review by AyushSrivastavaHyland — COMMENTED** [review]
Added some points to be added and modified

**Review by AyushSrivastavaHyland — APPROVED** [review]

File: process-services/latest/using/workflows.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+3. Select **Tenants**.
+
+4. Select **Create tenant**.
+
+      The **Create tenant** dialog box appears.
+
+5. Enter the following details as per your requirements.
+
+      * Enter the **Name** – The name of your tenant, for example, xxx.
```
Comment (anxumalo) [inline process-services/latest/using/workflows.md:30]: Add an example name here

File: process-services/latest/using/workflows.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+      * Specify the **Maximum allowed users in this tenant** - This is set at 10 bBy default, it is set at 10.
```
Comment (anxumalo) [inline process-services/latest/using/workflows.md:31]: Fix repetition as well as typo:
```suggestion
      * Specify the **Maximum allowed users in this tenant** - This is set at 10 by default, it is set at 10.

```

File: process-services/latest/using/workflows.md (hunk 4 above)
Comment (aritrihore) [reply process-services/latest/using/workflows.md:30]: Will the name 'hruser' do?

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**